psychopy/parallel/__init__.py

```diff
@@ -31,22 +31,32 @@
 import sys
 from psychopy import logging
 
-# To make life easier, only try drivers which have a hope in heck of working
+# To make life easier, only try drivers which have a hope in heck of working.
+# Because hasattr() in connection to windll ends up in an OSError trying to
+# load 32bit drivers in a 64bit environment, this passage is written as an
+# inconvenient try-except-construction.
 if sys.platform.startswith('linux'):
     from ._linux import PParallelLinux
     ParallelPort = PParallelLinux
 elif sys.platform == 'win32':
+    drivers = dict(inpout32=['_inpout', 'PParallelInpOut'],
+                   inpoutx64=['_inpout', 'PParallelInpOut'],
+                   dlportio=['_dlportio', 'PParallelDLPortIO'])
```

Comment:
I wonder whether we should use a dict of tuples instead of a dict of lists, since the values ought to be immutable?

Reply:
Yes, good idea.